Pipfile

```diff
@@ -5,6 +5,7 @@ name = "pypi"
 
 [packages]
 beautifulsoup4 = "==4.9.3"
+"google.cloud" = "*"
```

Comment:
Adding this will apply to all pipelines, perhaps only add it under your image's `requirements.txt` file

Reply:
@adlersantos - already Pipfile has "google.cloud" = "*"

Reply:
Let's remove this because this one's included in Cloud Composer

Reply:
done